app/metrics.yaml

```
@@ -130,3 +130,66 @@ settings_screen:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-08-17"
+
+shorcuts:
```

```suggestion
shortcuts:
```

I this typo in a couple places - is it intentional to evade a max length limitation?

---

typo + copy paste 😁 . I will change it.